1-mandatory/1-alarmclock/alarmclock.js

```diff
@@ -1,25 +1,46 @@
-function setAlarm() {}
-
-// DO NOT EDIT BELOW HERE
-
-var audio = new Audio("alarmsound.mp3");
-
-function setup() {
-  document.getElementById("set").addEventListener("click", () => {
-    setAlarm();
-  });
-
-  document.getElementById("stop").addEventListener("click", () => {
-    pauseAlarm();
-  });
-}
-
-function playAlarm() {
-  audio.play();
-}
-
-function pauseAlarm() {
-  audio.pause();
-}
-
-window.onload = setup;
+
+let countDown
+function setAlarm() {
+  const timeRemaining = document.getElementById("timeRemaining");
+  document.querySelector("body").style.backgroundColor = "white";
+  //  get the input value
+  let alarmSet = document.getElementById("alarmSet").value;
+  // get the element to show ouput
+  countDown = setInterval(()=> {
+ alarmSet--;
+  timeRemaining.innerText = `Time Remaining: 00:${alarmSet}`;
+   console.log(alarmSet);
+  if (alarmSet === 0) {
+    playAlarm();
+    document.querySelector("body").style.backgroundColor = "red";
+    clearInterval(countDown);
+  }}, 1000);
+}

```

Comment:
have a think what you might change in your innerText if you were to format it further - for example if someone input a time above 60 seconds, it would be nice if it formatted it into minutes & seconds